docker/test/integration/runner/dockerd-entrypoint.sh

```diff
@@ -8,8 +8,8 @@ echo '{
     "ip-forward": true,
     "log-level": "debug",
     "storage-driver": "overlay2",
-    "insecure-registries" : ["dockerhub-proxy.sas.yp-c.yandex.net:5000"],
-    "registry-mirrors" : ["http://dockerhub-proxy.sas.yp-c.yandex.net:5000"]
+    "insecure-registries" : ["dockerhub-proxy.dockerhub-proxy-zone:5000"],
```

Comment:
`dockerhub-proxy.dockerhub-proxy-zone`  seems not to be a valid URL, shall we set it in my local hosts file?

Reply:
If the host is not found it's just not used, AFAIR

Reply:
so does it mean the host is never used? but I see the log about trying to connect to the host but failed.

Reply:
does it say "served from upstream"? Or something like that?

Reply:
I got the following logs:
![image](https://user-images.githubusercontent.com/1518453/209604811-f28f1688-0bf3-44fa-8234-a5f80b4a8360.png)

Reply:
Lines 511-512